src/jsd/actuator_fsm_helpers.cc

```diff
@@ -535,10 +535,14 @@ fastcat::FaultType fastcat::Actuator::ProcessFaulted()
 
 fastcat::FaultType fastcat::Actuator::ProcessHalted()
 {
-  if (IsIdleFaultConditionMet()) {
-    ERROR("Act %s: %s", name_.c_str(), "Fault Condition present, faulting");
-    return ALL_DEVICE_FAULT;
-  }
+  //  Invoking a RESET will fail the following fault condition since the 
+  //  fault_code and emcy_error_code will contain the last-tripped fault information.
+  //  Currently, the only way for JSD to clear these codes is to call `EgdReset();
+  //
+  //if (IsIdleFaultConditionMet()) {
```

Comment:
Shouldn't we still monitor STO to fault. This state does not have an automatic transition.

It seems to me something like this should remain:

```
if (state_->actuator_state.sto_engaged) {
    ERROR("%s: STO Engaged", name_.c_str());
    ERROR("Act %s: %s", name_.c_str(), "Fault Condition present, faulting");
    return ALL_DEVICE_FAULT;
}
```

Reply:
not a bad idea, I'll check how this affects reset behavior when the STO is pressed with and without and provide a recommendation on Monday.